servicenow_connector.py

```diff
@@ -1400,53 +1456,62 @@ def _get_variables(self, param):
         request_params = {'request_item': sys_id}
 
         ret_val, auth, headers = self._get_authorization_credentials(action_result)
-        if (phantom.is_fail(ret_val)):
+        if phantom.is_fail(ret_val):
             return action_result.set_status(phantom.APP_ERROR, "Unable to get authorization credentials")
 
-        ret_val, response = self._make_rest_call_helper(action_result, endpoint, auth=auth, headers=headers, params=request_params)
+        ret_val, response = self._make_rest_call_helper(action_result, endpoint, auth=auth,
+                                headers=headers, params=request_params)
 
-        if (phantom.is_fail(ret_val)):
+        if phantom.is_fail(ret_val):
             return action_result.get_status()
 
         if not response.get('result'):
-            return action_result.set_status(phantom.APP_ERROR, 'No data found for the requested item having System ID: {0}'.format(sys_id))
+            return action_result.set_status(phantom.APP_ERROR, 'No data found for the \
+                        requested item having System ID: {0}'.format(sys_id))
```

Comment:
suggestion: Do you think we can fit all the params on the 2nd line?

Reply:
Yes I have made the suggested changes.